Assets/MRTK/Providers/Oculus/XRSDK/OculusXRSDKDeviceManager.cs

```diff
@@ -88,6 +88,22 @@ public override bool CheckCapability(MixedRealityCapability capability)
 
         #region Controller Utilities
 
+#if OCULUSINTEGRATION_PRESENT
+        /// <inheritdoc />
+        protected override GenericXRSDKController GetOrAddController(InputDevice inputDevice)
+        {
+            GenericXRSDKController controller = base.GetOrAddController(inputDevice);
+            OculusXRSDKTouchController oculusTouchController = controller as OculusXRSDKTouchController;
+
+            if(!oculusTouchController.IsNull())
+            {
+                oculusTouchController.UseMRTKControllerVisualization = cameraRig.IsNull();
+            }
+
+            return oculusTouchController;
```

Comment:
small nit on syntax: you could combine the cast and the null check as:

```suggestion
            if (controller is OculusXRSDKTouchController oculusTouchController)
            {
                oculusTouchController.UseMRTKControllerVisualization = cameraRig.IsNull();
            }

            return controller;
```

since `OculusXRSDKTouchController` doesn't represent a `UnityEngine.Object`, we don't have to do the special explicit `IsNull()` check.